tfhe/src/lib.rs

```diff
@@ -53,6 +53,7 @@ pub use js_on_wasm_api::*;
 mod test_user_docs;
 
 #[cfg(any(feature = "boolean", feature = "shortint", feature = "integer"))]
+/// cbindgen:ignore
```

Comment:
can't this be `//` instead ? I saw `///` where used for the same ignore but it makes them show up in doc
https://docs.rs/tfhe/0.1.12/tfhe/boolean/index.html#special-module-attributes

should probably be done later

Reply:
I know, cbindgen requires a doc comment, but we have a neat workaround as you can see above